.github/workflows/liveness-branch-sync.yml

```
@@ -0,0 +1,21 @@
+name: Liveness Release Branch Sync
+
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  branch-sync:
+    name: Branch Sync
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Let's pin the version:
```suggestion
      - name: Checkout repo
        uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0 https://github.com/actions/checkout/commit/24cb9080177205b6e8c946b17badbe402adc938f
```